utils/manifest-utils/src/keystore.rs

```diff
@@ -115,6 +115,7 @@ pub trait Keystore: KeystoreClone + Send {
     fn add(&mut self, add: &AddParams) -> anyhow::Result<AddResponse>;
     fn remove(&mut self, remove: &RemoveParams) -> anyhow::Result<RemoveResponse>;
     fn list(&self) -> Vec<Cert>;
+    fn verifier(&self, cert: &str) -> anyhow::Result<Box<dyn SignatureVerifier>>;
```

Comment:
Having dedicated method for creating verifier allows to nicely do sequence:
- verify Golem cert
- verify X509 cert
reject if failed to verify cert (which is done when building verifier)
- verify signature
reject if failed to verify signature

Reply:
Just realized that in case of node-descriptors Golem signature contains the cert, signature and signature algorithm values as single file, so this split will make sense only for x509, but it can be addressed when adding Golem cert support for audited payload.